Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -68,6 +68,39 @@ internal struct PaymentsRequest: APIRequest {
         try container.encodeIfPresent(data.checkoutAttemptId, forKey: .checkoutAttemptId)
     }
     
+    private struct AdditionalData: Encodable {
+        let allow3DS2: Bool?
+        let executeThreeD: Bool
+        
+        enum Configuration {
```

Comment:
We have similar naming for app to app redirects, including "native". I suggest giving more context to the `AdditionalData` namespace